<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="parasharb-w7" name="com.provenir.automation.framework.testcases.FacilityLevelWorkflow" tests="12" failures="1" timestamp="7 Dec 2015 05:27:55 GMT" time="173.109" errors="0">
  <testcase name="test15_verifyDefaultFacilityLevelWorkflow" time="10.220" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test16_verifysavedFacilityWorkflow" time="11.284" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test12_verifyDefaultCreditLevelWorkflow" time="0.117" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test10_goToHomePage" time="6.413" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test17_gotoHomePage" time="5.885" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test11_clickNewCreditRequest" time="3.144" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test19_verifyTaskManagementSection" time="1.711" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.provenir.automation.framework.helper.Credit360Helper.clickActionColumnAndVerifyAddedWorkflow(Credit360Helper.java:99)
	at com.provenir.automation.framework.testcases.FacilityLevelWorkflow.test19_verifyTaskManagementSection(FacilityLevelWorkflow.java:248)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- test19_verifyTaskManagementSection -->
  <testcase name="test14_clickTransactionInfoPage" time="16.858" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="landingPage" time="46.242" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test18_searchRequest" time="42.383" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test13_verifySavedWorkflow" time="17.510" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
  <testcase name="test20_gotoFacility360" time="11.342" classname="com.provenir.automation.framework.testcases.FacilityLevelWorkflow"/>
</testsuite> <!-- com.provenir.automation.framework.testcases.FacilityLevelWorkflow -->
